printer_mods/Iakabos/C920_mount/README.md

```diff
@@ -1 +1,18 @@
-Slim mount for logitech c920 webcam. Inspired by PhilLeMint's mount. Requires dissambly of webcam as seen here: https://www.ifixit.com/Guide/Logitech+C920+Webcam+Disassembly/115077
+# Slim Mount for Logitech C920 Webcam
+*Inspired by PhilLeMint's mount*
+
+## BOM
+
+Qty | Item
+:-- | :--
+1 | Logitech C920/C920s Webcam
+2 | M3 t-nuts
+2 | M3x8 SHCS
+
+## Instructions
+1. Disassemble webcam using instructions [seen here](https://www.ifixit.com/Guide/Logitech+C920+Webcam+Disassembly/115077) stopping after step 5
+2. Remove the metal arms can be removed from the factory supplied mount
```

Comment:
This sentence doesn't make sense.

Reply:
Agreed. Would suggest changing it to say "Remove the metal arms from the factory supplied mount"